developer-docs-site/docs/concepts/staking.md

```diff
@@ -58,7 +58,6 @@ As an owner:
 
 - Bob owns the funds that will be used for staking.
 - Only Bob can add or unlock or withdraw funds.
-- Only Bob can extend the lockup period.
```

Comment:
why remove this? the check on adding / withdrawing is the same as increasing lockup.

Reply:
Isn't lock up period now a governance-driven thing? I mean an owner can unlock anytime but the owner cannot extend lockup period because the lockup is extended automatically. The lockup period is set by the governance. So the above statement "Only Bob can extend the lockup period" is no longer true. Can you please confirm, @movekevin?

Reply:
The lockup period is automatically renewed when expired by the network. However, the owner can still "reset" the lockup. Example flow:

1. Alice wants to become a validator. She adds 100 coins for the stake.
2. Alice then requests to join the validator set when her node is ready. Once she’s added to the active validator set, a one-month lockup is automatically enforced on her total stake.
3. Alice’s stake lockup is renewed every month starting from her first added stake. This is similar to having your T-mobile subscription automatically renewed when the current billing cycle ends.
4. 25 days into the current one month cycle, a governance proposal is submitted with a voting duration of 7 days. Alice only has 5 days left in her current lockup. This is not enough to guarantee her stake will stay locked for the 7-day voting period. Alice wants to participate in voting on this proposal so she resets her lockup, which locks her stake for 30 days. She can now vote on the governance proposal.